### PR TITLE
[google_maps_flutter_platform_interface] Adds support for holes in polygon overlays to the Google Maps plugin

### PR DESCRIPTION
**Pull request by sanekyy**
Platform interface part of https://github.com/flutter/plugins/pull/1721

### CODE REVIEW

**Comment by fnicastri** [comment]
@cyanglaz ping

File: AUTHORS [hunk 1]
```diff
@@ -59,3 +59,5 @@ Kazuki Yamaguchi <y.kazuki0614n@gmail.com>
 Eitan Schwartz <eshvartz@gmail.com>
 Chris Rutkowski <chrisrutkowski89@gmail.com>
 Juan Alvarez <juan.alvarez@resideo.com>
+Aleksandr Yurkovskiy <sanekyy@gmail.com>
+
```
Comment (cyanglaz) [inline AUTHORS:63]: extra line

File: packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.0.5
```
Comment (cyanglaz) [inline packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md:1]: There is a non-breaking public api change, this should be 1.1.0

File: AUTHORS (hunk 1 above)
Comment (sanekyy) [reply AUTHORS]: fixed

File: packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md (hunk 2 above)
Comment (sanekyy) [reply packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md]: fixed

**Review by otopba — APPROVED** [review]

**Comment by fnicastri** [comment]
@sanekyy @otopba @cyanglaz 
Any news?

**Review by cyanglaz — APPROVED** [review]
LGTM

**Comment by otopba** [comment]
Hurrah🥳

**Pull request merged**